scripts/publish.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -e
+
+brew update && brew bump-formula-pr --tag=$(shell git describe --tags) --revision=$(shell git rev-parse HEAD) pxctest
```

Comment:
not sure how you normally publish to Homebrew (I do not know much about how this works)